test/functions/capitalize.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../lib/repetitivo/functions/capitalize";
+
+@include describe("capitalize") {
+  @include it("should uppercase first letter") {
+    @include assert-equal(capitalize("hello"), "Hello");
```

Comment:
Maybe test this with `"hello world"` → `"Hello world"` to avoid false positives?

Reply:
Done